app/components/elements/forms/submit_component.rb

```diff
@@ -4,8 +4,8 @@ module Elements
   module Forms
     # Component for a form submit button
     class SubmitComponent < ApplicationComponent
-      def initialize(form:, label: nil, variant: :primary, classes: [], **options)
-        @form = form
+      def initialize(form_id: nil, label: nil, variant: :primary, classes: [], **options)
```

Comment:
You really don't need a form to create a submit button. Setting the `form` attribute can be used to tie it to its form even if it isn't contained in the form.